src/Items.js

```diff
@@ -48,13 +48,19 @@ class Items extends React.Component {
     if (!items || !items.hasLoaded) return null;
     const itemRecords = items.records;
     const itemsFormatter = {
-      'Item: barcode': x => _.get(x, ['barcode']),
+      'Item: barcode': x => (
+        <span data-test-items-app-icon>
+          <AppIcon app="inventory" iconKey="item" size="small">
+            {_.get(x, ['barcode'])}
```

Comment:
For now, folio supports lodash tree shaking, which allows to import only necessary methods from lodash. Please, import get method as `import { get } from 'lodash'`

Reply:
@Yurii-Danylenko, eh, that lodash import hasn't changed in months. I agree we should implement `import { get } from 'lodash'` in new components, but I don't think we need to block PRs for existing components just because they don't make that update.

Reply:
@zburke, no props. Just wanted to apply Boy Scout Rule, to keep the code more or less up to date.

Reply:
I agree with both of you. It shouldn't block but I appreciate @Yurii-Danylenko's suggestions for making the code better. Since the change is so small I'll go ahead and make it.

Regarding testing of icons in the results list; I'll leave this for another time since it's not apart of the scope of this ticket. I hope that's cool :)